### PR TITLE
feat: add recent audit findings

### PR DESCRIPTION
**Pull request by deanamiel**
PR to publish recent reports and findings as per [AXE-2632](https://axelarnetwork.atlassian.net/jira/software/c/projects/AXE/boards/21?selectedIssue=AXE-2632)

[AXE-2632]: https://axelarnetwork.atlassian.net/browse/AXE-2632?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by milapsheth — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -6,44 +6,48 @@ found in repos, axelar-core, axelar-cgp-solidity, tofn, tofnd,
...
-| 2022-02 | NCC | `axelar-core` |
-| 2022-02 | [Ackee Blockchain](audits/2022-02%20Ackee%20blockchain.pdf) | `axelar-cgp-solidity` |
-| 2022-01 | NCC | `axelar-cgp-solidity` |
-| 2021-12 | Oak Security | `axelar-core`, `axelar-cgp-solidity` |
-| 2021-12 | [Cure53](audits/2021-12%20Cure53.pdf) | `axelar-core`, `axelar-cgp-solidity`, `tofn`, `tofnd`, `axelarjs-sdk` |
-| 2021-12 | NCC | `axelar-core`, `axelar-cgp-solidity` |
-| 2021-11 | [Adrian and Lucas](audits/2021-11%20Adrian%20and%20Lucas.pdf) | `tofn`, `tofnd` |
-| 2021-11 | [Commonprefix](audits/2021-11%20Commonprefix.pdf) | `axelar-cgp-solidity` |
-| 2021-08 | NCC | `tofn`, `tofnd` |
+|  Date   | Auditor                                                                               | Repos                                                                                            |                                                         Results                                                         |
```
Comment (milapsheth) [inline README.md:9]:
```suggestion
|  Date   | Audit                                                                               | Mitigations                                                                                            |                                                         Scope                                                         |
```
some rename suggestions

Comment (milapsheth) [inline README.md:9]: some reports are missing

Comment (deanamiel) [reply README.md]: implemented

Comment (deanamiel) [reply README.md]: all PDFs should be in there now, except code4rena because the final report has not been published yet

**Pull request merged**